ui/src/components/records/RecordFieldCell.svelte

```diff
@@ -57,9 +57,7 @@
     {:else if field.type === "file"}
         <div class="inline-flex">
             {#each CommonHelper.toArray(record[field.name]) as filename, i (i + filename)}
-                <figure class="thumb thumb-sm">
-                    <RecordFilePreview {record} {filename} />
-                </figure>
+                <RecordFilePreview {record} {filename} />
```

Comment:
Since now this will be always rendering a thumb, I think we can rename it to `RecordFileThumb` to be more clear.

We also will need to remove the `figure` wrapping inside the RecordUpsert panel > FileField (_you can skip the `class:fade` check_).